commerce/src/inline-price.js

```diff
@@ -145,9 +180,23 @@ export class HTMLPriceSpanElement extends HTMLSpanElement {
         );
         if (!options.wcsOsi.length) return false;
 
-        if (!this.placeholder.dataset.displayTax) {
-            // set default value for displayTax if not set neither in OST nor in price URL
-            options.displayTax = await this.resolveDisplayTax(service, options) || false;
+        if (!this.placeholder.dataset.displayTax || !this.placeholder.dataset.forceTaxExclusive) {
+            const [offerSelectors] = await service.resolveOfferSelectors(options);
+            const offers = selectOffers(await offerSelectors, options);
+            if (offers?.length) {
+                const { country, language } = options;
+                const offer = offers[0];
+                const [marketSegment = ''] = offer.marketSegments;
+
+                // set default value for displayTax if not set neither in OST nor in price URL
```

Comment:
I suggest using the term `merch link` instead of Price URL.

Reply:
fixed.